{"uuid":"0096a602-27c2-4473-bc12-ef0348e7f0e4","name":"helloworld","start":1544973830897,"stop":1544973913801,"children":["d32c338e-2ca4-43b4-a019-cd1a05eb3c18","f8d90c52-eada-4a38-8ef4-7624a7626b2e"],"befores":[{"name":"init","status":"passed","stage":"finished","description":"","start":1544973830938,"stop":1544973834407},{"name":"init","status":"passed","stage":"finished","description":"","start":1544973834408,"stop":1544973836642}],"afters":[{"name":"closeBrowser","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.net.ConnectException: Failed to connect to localhost/0:0:0:0:0:0:0:1:34741\nBuild info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:48'\nSystem info: host: 'MacBook-Pro-Vladislav.local', ip: 'fe80:0:0:0:10d1:f5da:ee35:1733%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.12.6', java.version: '11.0.1'\nDriver info: driver.version: RemoteWebDriver","trace":"org.openqa.selenium.WebDriverException: java.net.ConnectException: Failed to connect to localhost/0:0:0:0:0:0:0:1:34741\nBuild info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:48'\nSystem info: host: 'MacBook-Pro-Vladislav.local', ip: 'fe80:0:0:0:10d1:f5da:ee35:1733%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.12.6', java.version: '11.0.1'\nDriver info: driver.version: RemoteWebDriver\n\tat org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:92)\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:609)\n\tat org.openqa.selenium.remote.RemoteWebDriver.close(RemoteWebDriver.java:442)\n\tat LoginTestPopup.closeBrowser(LoginTestPopup.java:38)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)\n\tat org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:511)\n\tat org.testng.internal.Invoker.invokeConfigurations(Invoker.java:241)\n\tat org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)\n\tat org.testng.TestRunner.afterRun(TestRunner.java:820)\n\tat org.testng.TestRunner.run(TestRunner.java:592)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:398)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:392)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:354)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:302)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1145)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1067)\n\tat org.testng.TestNG.runSuites(TestNG.java:997)\n\tat org.testng.TestNG.run(TestNG.java:965)\n\tat org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)\n\tat org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)\nCaused by: java.net.ConnectException: Failed to connect to localhost/0:0:0:0:0:0:0:1:34741\n\tat okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:240)\n\tat okhttp3.internal.connection.RealConnection.connect(RealConnection.java:158)\n\tat okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:256)\n\tat okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:134)\n\tat okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:113)\n\tat okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)\n\tat okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)\n\tat okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)\n\tat okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)\n\tat okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)\n\tat okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)\n\tat okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)\n\tat okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)\n\tat okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:125)\n\tat okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)\n\tat okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)\n\tat okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)\n\tat okhttp3.RealCall.execute(RealCall.java:77)\n\tat org.openqa.selenium.remote.internal.OkHttpClient.execute(OkHttpClient.java:103)\n\tat org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:155)\n\tat org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)\n\t... 27 more\nCaused by: java.net.ConnectException: Connection refused (Connection refused)\n\tat java.base/java.net.PlainSocketImpl.socketConnect(Native Method)\n\tat java.base/java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:399)\n\tat java.base/java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:242)\n\tat java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:224)\n\tat java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:403)\n\tat java.base/java.net.Socket.connect(Socket.java:591)\n\tat okhttp3.internal.platform.Platform.connectSocket(Platform.java:125)\n\tat okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:238)\n\t... 47 more\n"},"stage":"finished","description":"","start":1544973913766,"stop":1544973913780},{"name":"closeBrowser","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.net.ConnectException: Failed to connect to localhost/0:0:0:0:0:0:0:1:18426\nBuild info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:48'\nSystem info: host: 'MacBook-Pro-Vladislav.local', ip: 'fe80:0:0:0:10d1:f5da:ee35:1733%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.12.6', java.version: '11.0.1'\nDriver info: driver.version: RemoteWebDriver","trace":"org.openqa.selenium.WebDriverException: java.net.ConnectException: Failed to connect to localhost/0:0:0:0:0:0:0:1:18426\nBuild info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:48'\nSystem info: host: 'MacBook-Pro-Vladislav.local', ip: 'fe80:0:0:0:10d1:f5da:ee35:1733%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.12.6', java.version: '11.0.1'\nDriver info: driver.version: RemoteWebDriver\n\tat org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:92)\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:609)\n\tat org.openqa.selenium.remote.RemoteWebDriver.close(RemoteWebDriver.java:442)\n\tat VacuumCleanerPageTest.closeBrowser(VacuumCleanerPageTest.java:57)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)\n\tat org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:511)\n\tat org.testng.internal.Invoker.invokeConfigurations(Invoker.java:241)\n\tat org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)\n\tat org.testng.TestRunner.afterRun(TestRunner.java:820)\n\tat org.testng.TestRunner.run(TestRunner.java:592)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:398)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:392)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:354)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:302)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1145)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1067)\n\tat org.testng.TestNG.runSuites(TestNG.java:997)\n\tat org.testng.TestNG.run(TestNG.java:965)\n\tat org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)\n\tat org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)\nCaused by: java.net.ConnectException: Failed to connect to localhost/0:0:0:0:0:0:0:1:18426\n\tat okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:240)\n\tat okhttp3.internal.connection.RealConnection.connect(RealConnection.java:158)\n\tat okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:256)\n\tat okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:134)\n\tat okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:113)\n\tat okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)\n\tat okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)\n\tat okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)\n\tat okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)\n\tat okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)\n\tat okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)\n\tat okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)\n\tat okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)\n\tat okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:125)\n\tat okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)\n\tat okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)\n\tat okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)\n\tat okhttp3.RealCall.execute(RealCall.java:77)\n\tat org.openqa.selenium.remote.internal.OkHttpClient.execute(OkHttpClient.java:103)\n\tat org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:155)\n\tat org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)\n\t... 27 more\nCaused by: java.net.ConnectException: Connection refused (Connection refused)\n\tat java.base/java.net.PlainSocketImpl.socketConnect(Native Method)\n\tat java.base/java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:399)\n\tat java.base/java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:242)\n\tat java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:224)\n\tat java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:403)\n\tat java.base/java.net.Socket.connect(Socket.java:591)\n\tat okhttp3.internal.platform.Platform.connectSocket(Platform.java:125)\n\tat okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:238)\n\t... 47 more\n"},"stage":"finished","description":"","start":1544973913793,"stop":1544973913797}]}